include/materials/NEMLStressBase.h

```diff
@@ -62,6 +62,11 @@ class NEMLStressBase : public ComputeStressBase
   /// Inelastic strain tensor
   MaterialProperty<RankTwoTensor> & _inelastic_strain;
 
+  const bool _compute_dt;
+  const Real _target_increment;
+  const MaterialProperty<RankTwoTensor> * _inelastic_strain_old;
+  MaterialProperty<Real> * _material_dt;
```

Comment:
are these pointers so that you can set them to nullptr when no compute_dt is defined?

Reply:
yes